charts/prometheus/templates/prometheus-config-configmap.yaml

```diff
@@ -210,6 +210,8 @@ data:
           # Pods have this label twice
           - regex: 'label_kubernetes_executor'
             action: labeldrop
+          - regex: 'label_kubernetes_pod_operator'
+            action: labeldrop
```

Comment:
@sjmiller609 @jpweber so these two drops affect only the ones that airflow adds itself, but ignore the ones that we add via our config file?

Reply:
it will drop any label that matches exactly `label_kubernetes_pod_operator` wherever it comes from. Same problem as the `label_kubernetes_executor` problem we had a few weeks back. I don't know what the further downstream impact is honestly.